src/components/notifications/ConfigurationTab.tsx

```diff
@@ -202,6 +239,86 @@ export class ConfigurationTab extends Component<{}, ConfigurationTabState> {
             )
     }
 
+    renderWebhookConfigurationTable() {
+        if (this.state.view === ViewType.LOADING) {
+            return (
+                <div className="flex" style={{ height: 'calc(100% - 68px)' }}>
```

Comment:
remove inline css if possible

Reply:
inline css used for particular piece of code, not used anywhere else

Reply:
done